greedybear/celery.py

```diff
@@ -86,4 +86,10 @@ def setup_loggers(*args, **kwargs):
         "schedule": crontab(minute=33),
         "options": {"queue": "default"},
     },
+    # run once every settings.DB_BACKUP_FREQUENCY * 24 hour
+    "db_backup": {
+        "task": "greedybear.tasks.db_backup",
+        "schedule": crontab(minute=0, hour=f'*/{24 * settings.DB_BACKUP_FREQUENCY}'),
```

Comment:
this is still wrong, hour and minute must be fixes, days should change

Reply:
Sir I currently had a doubt about the `days` parameter. Sir If you don't mind me asking, isn't the days parameter for 'day of the month' or 'day of the week'? I looked into a few cronjob syntaxes and it all suggested using either `hours` or `days=*/DAYS_INTERVAL`

Reply:
maybe we could just leave this to do it once every day

Reply:
Ok Sir.

I'm very new to celery and cron jobs, I'll try my best to learn it as soon as possible to be capable of working on Greedybear.

Reply:
Hello Sir, I just noticed that celery schedules support `datetime.timedelta` formats. I replaced the `crontab()` implementation with `timedelta(days=....)` implementation. Is this ok, or should we resort to settings daily cron jobs?

Reply:
where did you get this? have you tried it working?

Reply:
Sir I got that from the celery documentation.https://docs.celeryq.dev/en/stable/userguide/periodic-tasks.html?highlight=cron#available-fields

Reply:
I tried to test the cron job, but there seems to some problem. Maybe I'm running it incorrectly.

I'm not sure why but my tests dont print any logs from celery cron jobs. I've even tried adding `print()` to current cron job, and other cron jobs, but none of them seem to be running.

Reply:
why don't you just populate the `env_file` inside the `docker` folder with that DEBUG=True value?

anyway the logs should work and should appear in the `greedybear.log` file

Reply:
can you please explain which are your tests? cause you have implemented no tests here

Reply:
I've edited the env file this time and tried looking up the log files. the celery logs aren't present there.

> can you please explain which are your tests? cause you have implemented no tests here

Oh, I was actually referring to changing the celery schedules for all the cron jobs during debug. Just for testing if the cron jobs are working at all. It turns out the cron jobs are not working for some reason, or i'm looking for logs in the wrong place.

Reply:
It is difficult to guess from here why in your env celery does not work while in mine yes without any problems...in every case, there are no tests implemented for this thing. We need them so we can accept the PR

Reply:
Ok Sir. I'll try it. Also, are there any other ways to check if the celery cron job is running? or maybe manually trigger it?

Reply:
this are just some basic hints:

- You can set up a time that is close to yours if you want to see that the cron works
- you can use `.delay()` from `celery` to have a task called via celery. This excludes that there is a problem in celery but does not exclude that it could be one from the cronjob configuration
- a proper test will exclude bugs in normal launches, even without celery. So this is primary importance